src/content/docs/new-relic-control/agent-control/configuration.mdx

```diff
@@ -0,0 +1,284 @@
+
+To learn about all the settings in the configuration file, see [`values-newrelic.yaml`](https://github.com/newrelic/helm-charts/blob/master/charts/agent-control/values.yaml).
+
+## Sample configurations
+### Local configuration: New Relic infrastructure and Fluent Bit
+The following snippet demonstrates installing Agent Control with infrastructure monitoring and Fluent Bit for logs collection:
+
+<CollapserGroup>
+  <Collapser
+        id="agent-control-config"
+        title="Local config for infrastructure and Fluent Bit"
+      >
+    ```yaml
+    global:
+      cluster: "YOUR_CLUSTER_NAME"
+      licenseKey: "YOUR_LICENSE_KEY"
+      userKey: "YOUR_USER_KEY"
+
+    # See `https://github.com/newrelic/helm-charts/blob/master/charts/agent-control/values.yaml`
+    agent-control-deployment:
+      config:
+        fleet_control:
+          # Optional
+          # fleet_id: YOUR_FLEET_ENTITY_GUID
+          auth:
+            organizationId: "YOUR_ORGANIZATION_ID"
+        subAgents:
+          infrastructure:
+            type: newrelic/com.newrelic.infrastructure:0.1.0
+            content:
+              # Ref: `https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle`
+              # Recommended: check and define an explicit chart version (latest stable)
+              chart_version: "*"
+              
+              #chart_values:
+              #  newrelic-infrastructure:
+              #    enableProcessMetrics: true
+          logs:
+            type: newrelic/io.fluentbit:0.1.0
+            content:
+              # Ref: `https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging`
+              # Recommended: check and define an explicit chart version (latest stable)
+              chart_version: "*"
+
+              #chart_values:
+              #  newrelic-logging:
+              #    sendMetrics: true
+          agent-operator:
+            type: com.newrelic.k8s_agent_operator:0.1.0
+            chart_version: "*"
+    ```
+  </Collapser>
+
+</CollapserGroup>
+
+
+
+### Local configuration: OpenTelemetry
+The following snippet demonstrates installing Agent Control with OpenTelemetry and disabling the filelog receiver in the managed [`nr-k8s-otel-collector` Helm chart](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector#values):
+
+<CollapserGroup>
+  <Collapser
+    id="otel-config"
+    title="OpenTelemetry configuration"
+  >
+
+    ```yaml
+        global:
+          cluster: "YOUR_CLUSTER_NAME"
+          licenseKey: "YOUR_LICENSE_KEY"
+        # Values related to the agent control's Helm chart release.
+        # `https://github.com/newrelic/helm-charts/blob/master/charts/agent-control/values.yaml`
+        agent-control-deployment:
+          identityClientId: ""
```

Comment:
indentation is wrong

![Uploading Screenshot 2025-02-19 at 18.12.11.png…]()

Reply:
<img width="444" alt="Screenshot 2025-02-19 at 18 14 04" src="https://github.com/user-attachments/assets/73a23e25-c5e8-4b9b-bb6a-c8e710ce29f2" />